packages/sync/src/config/DataSyncConfig.ts

```diff
@@ -33,7 +32,7 @@ export interface DataSyncConfig {
    *
    * The storage you want your client to use (Uses window.localStorage by default)
    */
-  storage?: PersistentStore<PersistedData>;
+  storage?: LocalForage;
```

Comment:
do we have to change the type here? It should continue to be the `PersistentStore<PersistedData>` interface, and LocalForage is an implementation of this interface?